.grit/patterns/react.grit

```diff
@@ -1,9 +0,0 @@
-engine marzano(0.1)
-language js
-
-pattern ReactNode($name, $props, $children) {
```

Comment:
Why delete this?

Reply:
Not used anywhere, trivial pattern, might as well declutter while we can. Can add back if you prefer.

Reply:
Keep it. It's helpful for one offs.